docs/extensions/list.rst

```diff
@@ -0,0 +1,14 @@
+Sylius Extensions List
+======================
+
+.. note::
+
+    This section will grow whenever a new extension will be provided by the Sylius open-source community!
+
+The current list of Sylius Extensions presents as follows:
+
+* `Lakion\SyliusCmsBundle`_
```

Comment:
Shouldn't we use inline links here? There won't be the same extension mentioned twice and less lines = better :>